src/odemis/driver/actuator.py

```diff
@@ -111,7 +111,9 @@ def __init__(self, name, role, children, axes_map, ref_on_init=None, **kwargs):
         # position & speed: special VAs combining multiple VAs
         self.position = model.VigilantAttribute(self._position, readonly=True)
         for c, ax in children_axes.items():
-            def update_position_per_child(value, ax=ax, c=c):
+            def update_position_per_child(value, ax=None, c=c):
```

Comment:
That's an error. It completely changes the behaviour. Now ax is *always* `None`. This `ax=ax` is for storing the `ax` content per function, so it's a weird syntax, and have to be extra careful.